aten/src/ATen/native/TensorShape.cpp

```diff
@@ -3105,6 +3105,11 @@ Tensor view(const Tensor& self,
   return view_impl(self, size);
 }
 
+Tensor view_symint(const Tensor& self,
+            c10::SymIntArrayRef size) {
+  return view(self, c10::asIntArrayRefSlow(size));
```

Comment:
I think this should be `self.view(c10::asIntArrayRefSlow(size))`

You want to re-enter the dispatcher, so we can call XLA's implementation of view insttead of calling the one in this file.

Reply:
(insert plug for codegen'ing more symint kernels so we don't have to worry about getting these wrong 😛 )